src/renderers/WebGL1Renderer.js

```diff
@@ -1,7 +1,13 @@
 import { WebGLRenderer } from './WebGLRenderer.js';
 
-class WebGL1Renderer extends WebGLRenderer {}
+class WebGL1Renderer extends WebGLRenderer {
 
-WebGL1Renderer.prototype.isWebGL1Renderer = true;
+	constructor( parameters = {} ) {
+
+		super( { ...parameters, isWebGL1Renderer: true } );
```

Comment:
Can we do like in [`Vector3`](https://github.com/mrdoob/three.js/blob/dev/src/math/Vector3.js#L8) instead?

```js
WebGL1Renderer.prototype.isWebGL1Renderer = true;
```

Reply:
You are right, for some reason I tought it wouldn't work with a class, but it does.

Done!